src/main/java/jdbchomework/entity/AbstractEntity.java

```diff
@@ -39,10 +39,16 @@ public String getName() {
     }
 
     public void setName(String name) {
+        if (name == null || name.equals("")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)